src/main/cpp/t06_cinema.cpp

```diff
@@ -29,6 +29,50 @@
 using namespace std;
 
 int t06_cinema() {
-    
+     int n, m, y=0, k;
+    cin >> n >> m;
+    int a[20][20];
+    for(int i=0; i<n; i++)
+    {
+        for(int j=0; j<m; j++)
+        {
+            cin >> a[i][j];
+        }
+    }
+    cin >> k;
+    int j, c;
+    bool flag;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 'flag' can be reduced.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4488444)